README.md

```diff
@@ -54,15 +53,36 @@ Support for Node v8 is deprecated. Support will be officially dropped when it re
   * [Default mode](#default-mode)
     + [Default values](#default-values)
   * [Spectral configuration](#spectral-configuration)
-    + [Changing rule severity](#changing-rule-severity)
-    + [Custom rules](#custom-rules)
+    + [Changing Spectral rule severity](#changing-spectral-rule-severity)
+    + [Custom Spectral rules](#custom-spectral-rules)
 - [Warnings Limit](#warnings-limit)
 - [Turning off `update-notifier`](#turning-off-update-notifier)
 - [License](#license)
 
 <!-- tocstop -->
 
+## Getting Started
+The validator analyzes your API definition and reports any inconsistencies within the API definition. The validator is highly customizable, and supports both OpenAPI 3.0, OpenAPI 2.0 (Swagger 2.0), and Spectral rules.  You can easily create new rules and modify existing ones so that the tool will ensure consistency to your standards.
```

Comment:
```suggestion
The validator analyzes your API definition and reports any inconsistencies within the API definition. The validator is highly customizable, and supports both OpenAPI 3.0, OpenAPI 2.0 (Swagger 2.0), and Spectral rules.  You can easily create new rules and modify existing ones so that the tool will ensure compliance to your standards.
```